rocAL/rocAL/source/node_ssd_random_crop.cpp

```diff
@@ -101,44 +110,47 @@ void SSDRandomCropNode::update_node()
             //sample_option = 0;
             if (!sample_option)
             {
-                crop_box.x = 0;
-                crop_box.y = 0;
-                crop_box.h = in_height[i];
-                crop_box.w = in_width[i];
+                crop_box.l = 0;
+                crop_box.t = 0;
+                crop_box.r = 1;
+                crop_box.b = 1;
                 break;
             }
             for (int j = 0; j < _num_of_attempts; j++)
             {
                 x_drift_factor->renew();
                 float factor = 0.3f;
-                auto w_factor = factor + (x_drift_factor->get() * (1 - factor));
-                crop_box.w = w_factor * in_width[i];
+                w_factor = factor + (x_drift_factor->get() * (1 - factor));
```

Comment:
better to use local random parameters here rather than rali parameter factory like in random_bbox_reader

Reply:
Resolved